packages/netlify-cms-backend-fs/scripts/fs/fs-api.js

```diff
@@ -4,17 +4,19 @@ const chalk = require('chalk');
 const log = console.log;
 
 const { version, name } = require('../../package.json');
-log(chalk.green(`${name} (version: ${version})`));
+const packageLabel = `[${name}]`
+
+log(chalk.green(`${packageLabel} (version: ${version})`));
 
 const projectRoot = path.join(process.cwd());
-log(chalk.green(`Root path is ${ projectRoot }`));
+log(chalk.green(`\n${packageLabel} root path is ${projectRoot} \n`));
```

Comment:
Adds a couple new lines so it stands out a bit. Before it was being squished a bit and is unclear if this is coming from Gatsby or another lib: 

<img width="361" alt="screen shot 2019-01-23 at 5 20 25 pm" src="https://user-images.githubusercontent.com/236943/51647853-3f75f900-1f33-11e9-9a90-d505471398d1.png">

Reply:
I like this.